backend/.env.example

```diff
@@ -1,2 +1,4 @@
 DJANGO_SETTINGS_MODULE={{project_name}}.settings.local
 CELERY_BROKER_URL=amqp://broker:5672//
+# DATABASE_URL=postgres://{{project_name}}:password@db:5432/{{project_name}}
```

Comment:
I think we can add a comment above: 
`# Please choose postgres or sqlite as your DB:`